packages/Collapsible/stories/Collapsible.stories.js

```diff
@@ -1,9 +1,35 @@
 import React from "react";
-import { storiesOf } from "@storybook/react";
 import { withKnobs } from "@storybook/addon-knobs";
 
 import Variations from "./examples/Variations";
+import Showcase from "./examples/Showcase";
+import Props from "./Collapsible.mdx";
+import Collapsible from "../src";
 
-storiesOf("Collapsible", module)
-  .addDecorator(withKnobs)
-  .add("Variations", () => <Variations />);
+export default {
+  title: "Collapsible",
+  component: Collapsible,
+};
+
+export const showcase = Showcase;
+showcase.story = {
+  decorators: [withKnobs],
+  parameters: {
+    docs: { page: Props },
+    options: {
+      isToolshown: true,
+      showPanel: true,
+    },
+  },
```

Comment:
Can we add `viewMode: "story"` here? And really for all the non-mdx stories so we always get sent back to the canvas?

Reply:
Not needed anymore with new storybook config